openedx/features/content_type_gating/tests/test_partitions.py

```diff
@@ -0,0 +1,53 @@
+from datetime import date
+import ddt
+from mock import Mock, patch
+
+from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangolib.testing.utils import CacheIsolationTestCase
+from openedx.features.content_type_gating.partitions import create_content_gating_partition, CONTENT_GATING_PARTITION_ID
+from openedx.features.content_type_gating.models import ContentTypeGatingConfig
+from xmodule.partitions.partitions import UserPartitionError
+
+@ddt.ddt
```

Comment:
you don't need this or to import ddt since you're not currently using it

Reply:
Good call.